packages/search-engine/__tests__/debunce.test.ts

```diff
@@ -0,0 +1,28 @@
+import { debounce } from "../src/scripts/debounce";
+
+jest.useFakeTimers();
+describe('Debounce function', () => {
+    test('execute just once', () => {
```

Comment:
@simontorresdev We should bear on mind that tests description needs to follow BDD using the `given-when-then` approach.